[2022-11-29T07:49:15.343+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T07:47:51.849269+00:00 [queued]>
[2022-11-29T07:49:16.648+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T07:47:51.849269+00:00 [queued]>
[2022-11-29T07:49:16.661+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T07:49:16.670+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-29T07:49:16.684+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T07:49:17.128+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): task_6> on 2022-11-29 07:47:51.849269+00:00
[2022-11-29T07:49:18.995+0000] {standard_task_runner.py:55} INFO - Started process 259 to run task
[2022-11-29T07:49:19.112+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Can_I_change_name_to_this', 'task_6', 'manual__2022-11-29T07:47:51.849269+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp9qebr7qq']
[2022-11-29T07:49:19.150+0000] {standard_task_runner.py:83} INFO - Job 51: Subtask task_6
[2022-11-29T07:49:19.810+0000] {task_command.py:376} INFO - Running <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T07:47:51.849269+00:00 [running]> on host 68e922a5ec23
[2022-11-29T07:49:20.854+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Can_I_change_name_to_this
AIRFLOW_CTX_TASK_ID=task_6
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T07:47:51.849269+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T07:47:51.849269+00:00
[2022-11-29T07:49:20.881+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-29T07:49:20.888+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/tasks/data_cleansing_basic_stat.py']
[2022-11-29T07:49:20.960+0000] {subprocess.py:86} INFO - Output:
[2022-11-29T07:49:25.069+0000] {subprocess.py:93} INFO - {"message": "now perform data cleansing on lol_basic_stat", "time": "2022-11-29T07:49:25.068804"}
[2022-11-29T07:49:25.091+0000] {subprocess.py:93} INFO - {"message": "successfully perform data cleansing on lol_basic_stat", "time": "2022-11-29T07:49:25.087699"}
[2022-11-29T07:49:25.112+0000] {subprocess.py:93} INFO - {"message": "Now putting dataframe into pg and redshift", "time": "2022-11-29T07:49:25.100088"}
[2022-11-29T07:49:25.789+0000] {subprocess.py:93} INFO - {"message": "successfully put lol_basic_stat to pg and redshift", "time": "2022-11-29T07:49:25.788890"}
[2022-11-29T07:49:25.794+0000] {subprocess.py:93} INFO - {"message": "put lol_basic_stat to pg and redshift have used 0.9061301250003453s", "time": "2022-11-29T07:49:25.791534"}
[2022-11-29T07:49:26.227+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-29T07:49:26.424+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Can_I_change_name_to_this, task_id=task_6, execution_date=20221129T074751, start_date=20221129T074915, end_date=20221129T074926
[2022-11-29T07:49:26.602+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-29T07:49:27.131+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
